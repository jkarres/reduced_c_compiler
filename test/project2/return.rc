// testing return

function : int i() { return 10; }

function : float f() { return 1.23; }

function : bool b() { return true; }

function : void main() {
  cout << 1 << " " << i() << endl;
  cout << 2 << " " << f() << endl;
  cout << 3 << " " << b() << endl;
}

