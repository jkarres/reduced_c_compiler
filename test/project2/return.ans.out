1 10
2 1.23
3 true
